BetterSpecs/SpecObject.cs

```diff
@@ -70,11 +77,108 @@ public class Describe : SpecObject
     public class It : SpecObject
     {
         internal override void Before() => _ident = 8;
+        public Expect Expect { get; }
+        public It()
+        {
+            Expect = new Expect();
+        }
+
+        public Expect ExpectThatSubject()
+        {
+            return Expect;
+        }
+    }
+
+    public class Expect
```

Comment:
Give this class, why we need FluentAssertions. Doesn't make sense raise exception in methods?